datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyPartitionAssignmentStrategy.java

```diff
@@ -228,37 +251,39 @@ public StickyPartitionAssignmentStrategy(Optional<Integer> maxTasks, Optional<In
           .reduce((task1, task2) -> task1.getPartitionsV2().size() < task2.getPartitionsV2().size() ? task1 : task2)
           .get() : null;
 
-        // compute new assignment for that instance
-        Set<DatastreamTask> newAssignedTask = tasks.stream().map(task -> {
-          if (!dg.belongsTo(task)) {
-            return task;
-          }
-          boolean partitionChanged = false;
-          List<String> newPartitions = new ArrayList<>(task.getPartitionsV2());
-          Set<String> extraDependencies = new HashSet<>();
+      // compute new assignment for that instance
+      Set<DatastreamTask> newAssignedTask = tasks.stream().map(task -> {
```

Comment:
Sorry for my ignorance. I'm not able to find anything changing in the next few code diff blocks, except empty lines. Just want to be sure that I'm not missing anything.

Reply:
yeah sorry, there was an indentation issue in this part of the code, so just fixed that. next time i'll do this as part of a separate change.